official/wasm/go.mod

```diff
@@ -0,0 +1,234 @@
+module wasm
+
+go 1.21.1
+
+toolchain go1.21.6
```

Comment:
Maybe we can remove the `toolchain` line, using `1.21.1` should be fine:
```suggestion
```

Full disclosure, it's still a bit unclear to me when I would recommend a minimum dependency of Go `go 1.21.1` (language version) while using a newer toolchain (`toolchain go1.21.6`).